doc/Developing.md

```diff
@@ -12,7 +12,7 @@ The following instructions are for Linux and macOS.
 For Windows instructions [click here](#building-on-windows).
 
 ## Prerequisites
-* [Yarn package manager](https://yarnpkg.com/en/docs/install)
+* https://yarnpkg.com/en/docs/install
```

Comment:
That seems like a bad conflict resolution... the line removed is the one present on master